cmd/controller/disk.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import apis "github.com/openebs/node-disk-manager/pkg/apis/openebs.io/v1alpha1"
+
+// DiskInfo contains details of a disk which can be converted into api.Disk
+type DiskInfo struct {
+	ProbeIdentifiers ProbeIdentifier // ProbeIdentifiers contains some keys to uniquely identify each disk by probe
+	HostName         string          // HostName is hostName in which disk is attached
```

Comment:
HostName contains the node's hostname in which this disk is attached.

Reply:
done.